src/main/resources/assets/autorun/lang/en_us.json

```diff
@@ -1,5 +1,7 @@
 {
   "key.autorun.toggle": "Toggle Auto-Run",
+  "config.autorun.toggleRunOnStart": "Toggle Run On Start",
+  "config.autorun.toggleRunOnStart.description": "Should the game start with last key presed?",
```

Comment:
Same here, and also don't phrase the description as a question.
I suggest "Keep Auto-Run toggled between leaving and joining worlds or servers"